users/validators.py

```diff
@@ -21,10 +12,10 @@ def validate(rq_data):
         experience = rq_data.get('experience')
 
         if not isinstance(experience, int):
-            raise forms.ValidationError('Only integer values are supported')
+            raise PValidationError('Only integer values are supported')
```

Comment:
In the case when `rq_data` doesn't contain `experience`:
``` python
experience = rq_data.get('experience') # None
```
then:
``` python
not isinstance(experience, int) # True
```
as result will called:
``` python
raise PValidationError('Only integer values are supported')
```
Is it expected behavior here in the method `ProfileValidator.validate`?

Reply:
It's not related to this PR, but i want to accent on that